pkg/apis/core/types.go

```diff
@@ -5686,6 +5698,25 @@ type TopologySpreadConstraint struct {
 	// This is an alpha field and requires enabling MinDomainsInPodTopologySpread feature gate.
 	// +optional
 	MinDomains *int32
+	// NodeAffinityPolicy indicates how we will treat Pod's nodeAffinity/nodeSelector
+	// when calculating pod topology spread skew. Options are:
+	// - Honor: only nodes matching nodeAffinity/nodeSelector are included in the calculations.
+	// - Ignore: nodeAffinity/nodeSelector are ignored. All nodes are included in the calculations.
+	//
+	// If this value is nil, `Honor` will be used instead.
```

Comment:
```suggestion
	// If this value is nil, the behavior is equivalent to the Honor policy.
```

Similarly for NodeTaintsPolicy